mobile/src/main/java/org/horizontal/tella/mobile/media/MediaFileHandler.java

```diff
@@ -408,39 +410,38 @@ public static Single<VaultFile> importVideoUri(Context context, Uri uri, String
         } catch (Exception e) {
             FirebaseCrashlytics.getInstance().recordException(e);
             Timber.e(e, MediaFileHandler.class.getName());
-
             throw e;
         } finally {
             try {
                 retriever.release();
-            } catch (Exception ignore) {
-            }
+            } catch (Exception ignore) {}
         }
     }
 
+
     public static Single<VaultFile> importOthersUri(Context context, Uri uri, String parentId) throws Exception {
         String mimeType = context.getContentResolver().getType(uri);
 
-        try {
+        DocumentFile doc = DocumentFile.fromSingleUri(context, uri);
+        if (doc == null || !doc.exists()) {
+            throw new FileNotFoundException("File no longer exists at URI: " + uri);
```

Comment:
hola @wafalakhal :) Will we show this error message to the user if this fails or is this an Exception that we will log  later on? 

Can we make sure that if there is an error (in any step) we show an error to the user? (maybe you already did this, just double checking because I cannot reproduce it)

Reply:
if this is is a message that will be shown to the user, shouldn't it be translated?